lib/web/JsonpChunkTemplatePlugin.js

```diff
@@ -31,6 +31,16 @@ class JsonpChunkTemplatePlugin {
 				);
 				if (entries.length > 0) {
 					source.add(`,${JSON.stringify(entries)}`);
+				} else {
+					source.add(`,null`);
```

Comment:
This could be omitted when `chunk.getChildIdsByOrders().prefetch` is empty.

I don't want to increase chunk size if prefetching isn't used.

Also use `0` instead of `null`, that's shorter. (a bit)